ethminer/MinerAux.h

```diff
@@ -309,6 +309,22 @@ class MinerCLI
 				}
 			}
 		}
+                else if (arg == "--cuda-parallel-hash")
```

Comment:
I think you should also check if there is additional arg after it. See other cases checking `&& i + 1 < argc`.